tests/test_json_consistency.py

```diff
@@ -103,7 +104,10 @@ def check_all(checks_by_file):
             for key, checks in checks_by_key.items():
                 for function in checks:
                     error = function(filename, content, key)
-                    assert not error, error
+                    if error:
+                        error_context = f"\n{'=' * 80}\nFailed check : {function.__doc__} for key '{key}'\nFile: {filename}\n{'=' * 80}\n{error}"
```

Comment:
looks a bit strange to surface the error location that way. Wasn’t it easier to just throw the error from inside `function` ?